_posts/en/newsletters/2023-12-06-newsletter.md

```diff
@@ -0,0 +1,179 @@
+    different sets, it's possible to determine which set of chunks is
+    best for miners.
+
+    If the first set of chunks encompasses the current mempool and the
+    second set of chunks is what the mempool would look like after a new
+    transaction is included (and any necessary transactions are
+    evicted), then we can determine whether that transaction improves
+    the state of the mempool for miners.  This evidence-based evaluation
+    can replace several heuristics currently used in Bitcoin Core to
+    determine whether a transaction should be relayed, potentially
+    making it easier for wallets to use [CPFP][topic cpfp] and
+    [RBF][topic rbf] fee bumping, improving profitability for miners,
+    and simplifying some solutions to problems like [transaction
+    pinning][topic transaction pinning] (without creating new problems).
+
+    The WG archives can now be read by anyone, but only invited members
+    can post.  Some noteworthy topics they've discussed include:
+
+    - [Cluster mempool definitions and theory][clusterdef] defines the terms being
+      used in the design of cluster mempool.  It also describes a small
+      number of theorems that showcase some of the useful properties of
+      this design.  The single post in this thread (as of this writing) is
+      very useful in understanding other discussions by the WG.
+
+    - [Merging incomparable linearizations][cluster merge] looks at how
+      to merge two different sets of chunks ("chunkings") for the same
+      set of transactions, specifically _incomparable_ chunkings.
+      Chunkings can be compared if one of them always accumulates the
+      same or greater amount of fees within any number of vbytes
+      (discrete to the size of the chunks).  For example:
+
+      ![Comparable chunkings](/img/posts/2023-12-comparable-chunkings.png)
+
+      Chunkings are incomparable if one of them accumulates a greater
+      amount of fees within some number of vbytes but the other chunking
+      accumulates a greater amount of fees within a larger number of
+      vbytes.  For example:
+
+      ![Incomparable chunkings](/img/posts/2023-12-incomparable-chunkings.png)
+
+      As one of the theorems in the previously linked thread notes, "if
+      one has two incomparable chunkings for a graph, then another
+      chunking must exist which is strictly better than both".  That
+      means an effective method for merging two different incomparable
+      chunkings can be a powerful tool for improving miner
+      profitability.  For example, miners Alice and Bob could both try
+      to find the optimal chunking and could share their results with
+      each other in a way that would guarantee they'd both obtain a
+      third chunking that was at least as good as either of their
+      original chunkings---and possibly better.
+
+    - [Post-cluster package RBF][cluster rbf] discusses an alternative
```

Comment:
you might want to note we don't even *have* package rbf today, and what is proposed pre-cluster mempool is very limited